.github/workflows/main.yml

```diff
@@ -84,8 +84,9 @@ jobs:
       - uses: actions/checkout@v4
       - name: Install packages
         run: |
+          sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```

Comment:
Compilers are getting stricter by the day, so more recent can help learn of issues before users.

Reply:
Hi @hartwork , thank you for your contribution!

* You're right: we do want to move to newer compilers. However, it is a real downside that the Ubuntu 22.04 Github Actions runner images do not have `gcc-13` by default anymore and that we'll have to wait until (probably) August before Ubuntu 24.04 becomes General Availability.
* The line `sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y` was actually included by the Github Runner Images in the first place, so there's no additional impact in having here
* The Python build for Linux is indeed still running `gcc-12` AFAICT. Going back there would make sense, but I believe that is not priority as it is now.
* Since NumPy has migrated to Ubuntu 24.04 [beta] in https://github.com/numpy/numpy/pull/26447 , I propose doing the same after the PGM meet-up is finished and Automatic Tap ( #74 ) is closed.